Tasks/Dubinin/Task1/Task1.py

```diff
@@ -0,0 +1,9 @@
+#Initial Data
+Amount = 20000
+Year = 5
+IR_y = 0.15
```

Comment:
these values should be of the decimal data type, since they are money-related, -1 point for the unwise choice